lib/parser.js

```diff
@@ -84,14 +84,29 @@ function summary (scanner) {
     node.children.push(t)
   }
 
-  // ... "(" <scope> ")" ...
-  let s = scope(scanner)
-  if (s instanceof Error) {
+  // ... <scopeStart> <scope> <scopeEnd> ...
+  let ss = scopeStart(scanner)
```

Comment:
I would be tempted to pull `scope-start` an `scope-end` into the `scope` parser, replacing the current logic that just ignores `(` and `)`.

Reply:
I thought the same thing, and did so at first, the issue was I didn't see an example of returning multiple tokens.  I can have it return an array or error, but that would be a new api in here.  Is that the direction we want to go?